package.json

```diff
@@ -31,6 +31,7 @@
   "dependencies": {
     "blueimp-md5": "2.18.0",
     "commander": "6.2.1",
+    "d3-scale": "3.2.3",
```

Comment:
according to the CI results, you also need to push package-lock.json

Reply:
Ah thanks!